pkg/e2e/compose_build_test.go

```diff
@@ -66,6 +66,21 @@ func TestLocalComposeBuild(t *testing.T) {
 		res.Assert(t, icmd.Expected{Out: `"FOO": "BAR"`})
 	})
 
+	t.Run("build with multiple build-args ", func(t *testing.T) {
+		for i := 0; i < 5; i++ {
+			// ensure local test run does not reuse previously build image
+			c.RunDockerOrExitError("rmi", "-f", "multiargs")
```

Comment:
Your test works, it’s just that here it’s
```suggestion
		c.RunDockerOrExitError("rmi", "-f", "multi-args_multiargs")
```